### DISCUSSION TITLE
Make shipping tab separate block in BO order view page

### DISCUSSION THREAD

**Discussion by ks129** [PrestaShop/PrestaShop#41075] [Ideas]
#### Is your feature request related to a problem?

If you have a huge amount of orders, every extra click you have to make just waste time. And shipping tab is one of such cases where you currently waste time by clicking.

#### Describe the solution you'd like

Put shipping information in a separate block.

#### Alternatives you've considered

Make shipping tab default, but changing status is also important, so this might not be the best idea.

**Comment by hibatallahAouadni** [comment]
Hello @ks129 

Thank you for your suggestion. The Product Team will take it into consideration for future developments. 
Please be aware that there is no guarantee that this feature will be developed anytime soon. If this is important to you, we strongly suggest you consider [hiring a specialist](https://www.prestashop.com/en/experts) to help you create it. 

If you are willing to implement this feature into PrestaShop as a [Pull Request](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/), the teams will be happy to help you by ensuring that it is appropriate and coherent with the rest of the project. If that is the case, please let us know! 

Thank you
